content/docs/glossary/multisig.mdx

```diff
@@ -158,7 +158,7 @@ Company account setup:
 <CodeExample>
 
 ```
-  Master Key Weight: 0 (Turned off so this account can't do anything without an employee)
+  Master Key Weight: 0 (Turned off so this account can't do anything without an employee.  Make sure to set this weight *last* so you don't lock the account before setting up the other signers.)
```

Comment:
It actually doesn't need to be the last, it can be in the transaction that adds other signers. I'm not sure this comment is necessary.

Reply:
But you're correct that you can't set the master weight to zero then in a separate transaction go ahead and add a signer.

Reply:
You're right.  I think the Alert above does enough to warn people, so it's fine without this line.  I'm getting rid of it, then merging.  Thanks for your input!